virtio-devices/src/balloon.rs

```diff
@@ -392,6 +403,20 @@ impl VirtioDevice for Balloon {
         self.read_config_from_slice(self.config.lock().unwrap().as_slice(), offset, data);
     }
 
+    fn write_config(&mut self, offset: u64, data: &[u8]) {
+        // The "actual" field is the only mutable field
+        if offset != self.config_actual_offset || data.len() != self.config_actual_size {
```

Comment:
These values will never change (otherwise that would break the kernel backward compatibility), which is why you can simply provide their hardcoded value here. `offset` will always be 4 and `size` should be 4 as well.